External/Plugins/ResultsPanel/Helpers/ResultsPanelHelper.cs

```diff
@@ -1,83 +1,249 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
 using PluginCore;
 using PluginCore.Localization;
 using PluginCore.Managers;
 using WeifenLuo.WinFormsUI.Docking;
 
 namespace ResultsPanel.Helpers
 {
-    class ResultsPanelHelper
+    internal static class ResultsPanelHelper
```

Comment:
I have mixed feelings about making this fully static, but I don't think I will make it a blocker.